content/en/docs/plugins/kotlin-plugin/features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Features"
+images: []
+menu:
+docs:
+parent: "kotlin-plugin"
+identifier: "kotlin-plugin-features"
+weight: 10
+---
+
+To help you take full advantage of the concise, safe, and pragmatic nature of Kotlin, Fixture Monkey provides a Kotlin plugin.
+- Using `PrimaryConstructorArbitraryIntrospector` as the default introspector to generate Kotlin classes with its primary constructor.
+- Fixture Monkey Extension Functions
+- Kotlin DSL Exp
+
+## Dependencies
+##### fixture-monkey-kotlin
+#### Gradle
+```groovy
+testImplementation("com.navercorp.fixturemonkey:fixture-monkey-kotlin:0.6.2")
+```
+
+#### Maven
+```xml
+<dependency>
+  <groupId>com.navercorp.fixturemonkey</groupId>
+  <artifactId>fixture-monkey-kotlin</artifactId>
+  <version>0.6.2</version>
+  <scope>test</scope>
+</dependency>
+```
+
+##### fixture-monkey-starter-kotlin
+
+To help you get started using Fixture Monkey in a Kotlin environment, there is also a starter dependency **fixture-monkey-kotlin-starter** that comes with pre-configured dependencies such as lombok or fixture-monkey-jakarta-validation.
```

Comment:
I think lombok should be removed. It looks to do not provides lombok dependencies.

Reply:
![Screen Shot 2023-09-12 at 1 36 52](https://github.com/naver/fixture-monkey/assets/24649991/4e83ac85-819b-4c44-b7e7-99fc1085c22d)